Makefile

```
@@ -6,6 +6,7 @@ prepare:
 
 serve:
 	hugo server \
+        --bind 0.0.0.0 \
```

please fix the indentation 
```suggestion
		--bind 0.0.0.0 \
```

---

Fixed